### ISSUE TITLE
Custom number formater

### ISSUE THREAD

**Issue by patkwee** [cocopon/tweakpane#198]
It would be great to be able to provide a custom formatter function at addInput, e.g. with the following interface:

```javascript
pane.addInput( PARAM, "mynumber", { formatter: x => x.toFixed(5) } );
```
This would allow to use custom number formatting, e.g. with libraries like d3-format, to format very small or very large numbers.
At the moment I am 'hacking' this via:

```javascript
const input = pane.addInput( PARAM, "mynumber" );
input.controller.controller.view.formatter = d3.format(".3g");
input.controller.controller.view.formatter_ = d3.format(".3g");
input.controller.controller.view.update();
```

**Comment by cocopon** [comment]
Implemented in #200. Please wait for the next release.

![image](https://user-images.githubusercontent.com/602961/111852205-6e445a00-8959-11eb-96a6-0963ca5cf67d.png)

**Comment by patkwee** [comment]
Great! Thank you very much.

**Comment by cocopon** [comment]
Released in 2.1.2. Thank you for your opinion :)
https://cocopon.github.io/tweakpane/input.html#number_formatter
